### DISCUSSION TITLE
[Support] Cannot use 'in' operator to search for 'enable' in egg-cors

### DISCUSSION THREAD

**Discussion by SuJunming** [midwayjs/midway#3795] [Q&A]
### Describe the problem（描述问题）

运行报错如下:
<img width="1036" alt="image" src="https://github.com/midwayjs/midway/assets/13981432/41e32a31-2cd3-47cc-b536-c27b1588b35c">
<img width="451" alt="image" src="https://github.com/midwayjs/midway/assets/13981432/5f2f175a-8563-4189-bdac-74fad20288a8">
<img width="235" alt="image" src="https://github.com/midwayjs/midway/assets/13981432/69970ef5-f268-45ee-bcf9-28a0c5b7725b">

### Midway Versions（Midway 版本）

<img width="555" alt="image" src="https://github.com/midwayjs/midway/assets/13981432/9f9a18ab-b789-4774-b413-4eb36842eaf2">

**Comment by SuJunming** [comment]
本来找不到midwayjs/decorator，但重新安装后：就是现在的错误

**Reply by czy88840616** [reply]
egg 的问题 用 ts 起必须用 ts-node

**Reply by SuJunming** [reply]
没有找到3的ts-node，是怎么启动
现在：    "dev": "cross-env ets && cross-env NODE_ENV=local mwtsc --watch --run @midwayjs/mock/app.js",

**Reply by czy88840616** [reply]
用 mwtsc 不应该走到 ts 里的，因为走的是编译后的 js，检查看看哪里走到了 ts 中。

**Reply by SuJunming** [reply]
egg换koa方便吗？是否只是Configuration里的imports

**Reply by czy88840616** [reply]
如果没有用到egg插件的话仅仅只是包/配置换一下

**Reply by Hyperionlucky** [reply]
我也是新建的项目，jtest run是没有问题的，但是vscode jest debug过程中，出现了这个问题，我的launch.json配置如下,这种要怎么解决
![image](https://github.com/midwayjs/midway/assets/44837878/0bc67d89-5423-4ed2-9417-1903a0f1b68f)

**Reply by waitingsong** [reply]
你试试看把单测命令行写到 package.json 的 script 中，然后在 launch.json 里面写成 npm run 的格式，
比如
```
      "runtimeArgs": [
        "run",
        "test",
        "--",
        "--runInBand"
      ],
      "runtimeExecutable": "npm",
```

**Reply by Hyperionlucky** [reply]
这个问题我刚刚解决了，按照我那样的写法单测的debug是可以通过的，我之前出问题是点击了jtest runner在代码中的插件按钮，如下：
![image](https://github.com/midwayjs/midway/assets/44837878/266bbe8d-7f07-449d-ac03-2170038c4031)
原因是这个debug插件按钮并没有走vscode launch.json的runtimeArgs，需要自己在setting.json里对这个插件的debugOptions配置与launch.json的runtimeArgs相近的内容，关键还是要用ts-node去起jest，配置如下：
![image](https://github.com/midwayjs/midway/assets/44837878/26d2a56a-59cd-4fec-bbb9-64d338e56137)

**Reply by waitingsong** [reply]
haha 。 我是用 mocha 的，没用过这个插件。

**Reply by Hyperionlucky** [reply]
哈哈哈哈哈好的，而且我发现咱们文档上写的jest配置应该是针对macos的，没有考虑windows，windows读取`${workspaceRoot}/node_modules/.bin/jest` 也会报错，需要对所有涉及到`.bin/jest` 的配置都改成`/jest/bin/jest.js`

**Reply by waitingsong** [reply]
czy88840616  的开发机是苹果 😄

**Reply by czy88840616** [reply]
而且还没有windows...之前为了开发自费了一年pd，也没用几次，太不划算了。
